app/controllers/api/v1/sharing/folders_controller.rb

```diff
@@ -6,15 +6,16 @@ def index
   end
 
   def show
-    render json: current_user.folder_seach(search_params).children
+     folder_contents = current_user.folder_search(input_params[:id])
```

Comment:
Like that you are scoping to the user. Looks like the indentation is a little funny.